packages/arb-ts/token_lists/generate_json.ts

```diff
@@ -0,0 +1,126 @@
+import { instantiateBridge } from '../scripts/instantiate_bridge'
+import { providers, utils, Wallet, BigNumber, constants, ethers } from 'ethers'
+import { StandardArbERC20__factory } from '../src/lib/abi/factories/StandardArbERC20__factory'
+import { ERC20__factory } from '../src/lib/abi/factories/ERC20__factory'
+
+import { TokenInfo, TokenList } from './tokenListTypes'
+import { writeFileSync } from 'fs'
+import axios from 'axios'
+
+const gen = async () => {
+  const tokens: TokenInfo[] = []
+
+  const { bridge, l1Network, l2Network } = await instantiateBridge()
+  const gatewaySetData = await bridge.getL2GatewaySetEventData()
+  // flatten in case they were set more than once
+  const tokenAddresses = [...new Set(gatewaySetData.map(data => data.l1Token))]
+  const logoUris = (
```

Comment:
instead of querying an external api for these URI assets, can't we just use something like
`https://github.com/trustwallet/assets/blob/825cf868f7bfdb37664040ab0b676b80bb342f49/blockchains/ethereum/assets/${ L1 token address }/logo.png`

Reply:
then we can drop the axios dependency

Reply:
hm, only thing is i'd want to verify the logo is indeed present, and now we're back to doing a bunch of external queries?